drivers/i2c/i2c_ite_enhance.c

```diff
@@ -734,20 +720,19 @@ static const struct i2c_driver_api i2c_enhance_driver_api = {
 		     (DT_INST_PROP(inst, clock_frequency) ==                    \
 		     I2C_BITRATE_FAST_PLUS), "Not support I2C bit rate value"); \
 	static void i2c_enhance_config_func_##inst(void);                       \
-	static const struct i2c_alts_cfg                                        \
-		i2c_alts_##inst[DT_INST_NUM_PINCTRLS_BY_IDX(inst, 0)] =         \
-			IT8XXX2_DT_ALT_ITEMS_LIST(inst);                        \
 										\
 	static const struct i2c_enhance_config i2c_enhance_cfg_##inst = {       \
 		.base = (uint8_t *)(DT_INST_REG_ADDR(inst)),                    \
 		.irq_config_func = i2c_enhance_config_func_##inst,              \
 		.bitrate = DT_INST_PROP(inst, clock_frequency),                 \
 		.i2c_irq_base = DT_INST_IRQN(inst),                             \
 		.port = DT_INST_PROP(inst, port_num),                           \
-		.alts_list = i2c_alts_##inst,                                   \
 		.gpio_dev = DEVICE_DT_GET(DT_INST_PHANDLE(inst, gpio_dev)),     \
+		.scl_pin = DT_INST_GPIO_PIN(inst, scl_gpios),                   \
+		.sda_pin = DT_INST_GPIO_PIN(inst, sda_gpios),                   \
```

Comment:
please use `gpio_dt_spec` facilities to collect gpio cells (and _dt APIs as well)

Reply:
Done.